<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_main">
      <property name="styleSheet">
       <string notr="true">border-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="Tab_Orbit">
          <attribute name="title">
           <string>Orbit</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QFrame" name="frame_input">
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>500</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="7" column="1">
               <widget class="QDateTimeEdit" name="dateTimeEdit_epoch1">
                <property name="dateTime">
                 <datetime>
                  <hour>6</hour>
                  <minute>0</minute>
                  <second>0</second>
                  <year>1999</year>
                  <month>9</month>
                  <day>20</day>
                 </datetime>
                </property>
                <property name="date">
                 <date>
                  <year>1999</year>
                  <month>9</month>
                  <day>20</day>
                 </date>
                </property>
                <property name="time">
                 <time>
                  <hour>6</hour>
                  <minute>0</minute>
                  <second>0</second>
                 </time>
                </property>
                <property name="displayFormat">
                 <string>yyyy-MM-dd HH:mm</string>
                </property>
                <property name="calendarPopup">
                 <bool>false</bool>
                </property>
                <property name="timeSpec">
                 <enum>Qt::UTC</enum>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_inc">
                <property name="text">
                 <string>Inclination (i) [deg]</string>
                </property>
               </widget>
              </item>
              <item row="7" column="0">
               <widget class="QLabel" name="label_epoch1">
                <property name="text">
                 <string>Initial Epoch (UTC)</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="lineEdit_inc">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QLineEdit" name="lineEdit_nu">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_a">
                <property name="text">
                 <string>21000</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit_ecc">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLineEdit" name="lineEdit_argp">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_a">
                <property name="text">
                 <string>Semi-major axis (a) [km]</string>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_nu">
                <property name="text">
                 <string>True anomaly (ν) [deg]</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_raan">
                <property name="text">
                 <string>RAAN [km]</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_ecc">
                <property name="text">
                 <string>Eccentricity (e) [-]</string>
                </property>
               </widget>
              </item>
              <item row="8" column="1">
               <widget class="QDateTimeEdit" name="dateTimeEdit_epoch2">
                <property name="displayFormat">
                 <string>yyyy-MM-dd HH:mm</string>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_argp">
                <property name="text">
                 <string>Argument of periapsis (ω) [deg]</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_tag">
                <property name="text">
                 <string>Orbit</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="lineEdit_raan">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_tag">
                <property name="text">
                 <string>Tag</string>
                </property>
               </widget>
              </item>
              <item row="8" column="0">
               <widget class="QLabel" name="label_epoch2">
                <property name="text">
                 <string>Final Epoch (UTC)</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Tab_Station">
          <attribute name="title">
           <string>Station</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QFrame" name="frame">
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QCheckBox" name="checkBox_station">
                <property name="text">
                 <string>Existing station?</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_station_tag">
                <property name="font">
                 <font>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Tag</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_station_tag">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Madrid</string>
                </property>
                <property name="dragEnabled">
                 <bool>false</bool>
                </property>
                <property name="readOnly">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Latitude</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit_station_coord_lat">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>40.416729</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Longitude</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="lineEdit_station_coord_long">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>-3.703339</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Tab_Output">
          <attribute name="title">
           <string>Output</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QFrame" name="frame_2">
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QCheckBox" name="checkBox_orbitview">
                <property name="text">
                 <string>3D Orbit View</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QCheckBox" name="checkBox_groundtrack">
                <property name="text">
                 <string>Ground Track</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QCheckBox" name="checkBox_ephem">
                <property name="text">
                 <string>Ephemerides</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="comboBox_groundtrack_select">
                <property name="duplicatesEnabled">
                 <bool>false</bool>
                </property>
                <item>
                 <property name="text">
                  <string>2D</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>3D</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_buttons">
         <property name="maximumSize">
          <size>
           <width>1000</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_cancel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Cancel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_reset">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Reset all</string>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_execute">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Execute</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_station</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_station_tag</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_station</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_station_coord_lat</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_groundtrack</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_groundtrack_select</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>234</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>376</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_tag</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_a</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_ecc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_argp</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_raan</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_inc</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_nu</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_station_tag</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_reset</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_station_coord_lat</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_station</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_station_coord_long</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>113</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
